django/forms/models.py

```diff
@@ -347,7 +347,16 @@ def _update_errors(self, errors):
         # Override any validation error messages defined at the model level
         # with those defined at the form level.
         opts = self._meta
-        for field, messages in errors.error_dict.items():
+
+        # Relates to #24706 - allowing models generated by
```

Comment:
Oops, I read it as "ticket reference not needed" and merged with the comment. At least for me, I think it's helpful to point out at least one use case as that code doesn't seem self-explanatory.